server/app/controllers/fusor/api/v21/deployments_controller.rb

```diff
@@ -64,12 +64,14 @@ def deploy
       end
     end
 
-    def validate
-      status = @deployment.valid? ? 200 : 422
-
-      render status: status, json: {
-        :errors => @deployment.errors.full_messages,
-        :warnings => @deployment.warnings
+    def validation
```

Comment:
Why the change in action name? Typically actions are verbs (create, destroy, etc).